locales/es/common.json

```diff
@@ -119,16 +118,12 @@
   "Story Teller": "Cuentacuentos",
   "Success! Your account has been created.": "Ha sido un éxito. Su cuenta ha sido creada.",
   "Systems": "Sistemas",
-  "TIME.days_many": "{{count}} días",
```

Comment:
There's definitely something off here. It seems that the i18next-parser was not able to identify the correct plural forms per language for your run.

Because of this, we are losing the `_many` plural keys that we already had for the relevant languages.

Eg. from https://jsfiddle.net/6bpxsgd4
![image](https://github.com/ForgeVTT/forge-translations/assets/34374883/49ac9b8b-7748-4049-9ab4-6d68e6c5d0f0)

I'd recommend making sure that the `i18next-parser` is up to date (6.5.0 seems to be the latest), but I'm not certain what might account for this behaviour on your env, since it should automatic detect that that language uses `_many`